modules/installation-aws-permissions.adoc

```diff
@@ -9,287 +9,172 @@ When you attach the `AdministratorAccess` policy to the IAM user that you create
-|`elasticloadbalancing:DescribeTargetGroups`
-|Yes
-|Updating LB targets for current servers
-
-|`openshift-machine-api`
-|`elasticloadbalancing:RegisterTargets`
-|Yes
-|Updating LB targets for current servers
-
-|`openshift-image-registry`
-|`iam:GetUser`
-|No
-|Fetching self
-
-|`openshift-image-registry`
-|`s3:CreateBucket`
-|Yes
-|Registry self-creates bucket
-
-|`openshift-image-registry`
-|`s3:DeleteBucket`
-|Yes
-|Registry self-creates bucket
-
-|`openshift-image-registry`
-|`s3:PutBucketTagging`
-|Yes
-|Registry self-creates bucket
-
-|`openshift-image-registry`
-|`s3:GetBucketTagging`
-|Yes
-|Registry self-creates bucket
-
-|`openshift-image-registry`
-|`s3:PutEncryptionConfiguration`
-|Yes
-|Registry self-creates bucket
-
-|`openshift-image-registry`
-|`s3:GetEncryptionConfiguration`
-|Yes
-|Registry self-creates bucket
-
-|`openshift-image-registry`
-|`s3:PutLifecycleConfiguration`
-|Yes
-|Registry self-creates bucket. Cleans up multipart uploads
-
-|`openshift-image-registry`
-|`s3:GetLifecycleConfiguration`
-|Yes
-|Registry self-creates bucket. Cleans up multipart uploads
-
-|`openshift-image-registry`
-|`s3:GetBucketLocation`
-|Yes
-|Working with registry host
-
-|`openshift-image-registry`
-|`s3:ListBucket`
-|Yes
-|Working with registry bucket
-
-|`openshift-image-registry`
-|`s3:HeadBucket`
-|Yes
-|Working with registry bucket
-
-|`openshift-image-registry`
-|`s3:GetObject`
-|Yes
-|Working with registry bucket
-
-|`openshift-image-registry`
-|`s3:PutObject`
-|Yes
-|Working with registry bucket
-
-|`openshift-image-registry`
-|`s3:DeleteObject`
-|Yes
-|Working with registry bucket
-
-|`openshift-image-registry`
-|`s3:ListBucketMultipartUploads`
-|Yes
-|Working with registry bucket
-
-|`openshift-image-registry`
-|`s3:AbortMultipartUpload`
-|Yes
-|Working with registry bucket
-
-|`openshift-image-ingress`
-|`iam:GetUser`
-|No
-|Fetching self
-
-|`openshift-image-ingress`
-|`elasticloadbalancing:DescribeLoadBalancers`
-|Yes
-|Ingress creates and removes names associated with ELB
-
-|`openshift-image-ingress`
-|`route53:ListHostedZones`
-|Yes
-|Ingress creates and removes names associated with ELB
-
-|`openshift-image-ingress`
-|`route53:ChangeResourceRecordSets`
-|Yes
-|Ingress creates and removes names associated with ELB
-
-|`openshift-image-ingress`
-|`tag:GetResources`
-|Yes
-|Used to find appropriate load balancer
-
-|===
+.Required EC2 permissions for installation
+* `ec2:AllocateAddress`
+* `ec2:AssociateAddress`
+* `ec2:AssociateDhcpOptions`
+* `ec2:AssociateRouteTable`
+* `ec2:AttachInternetGateway`
+* `ec2:AuthorizeSecurityGroupEgress`
+* `ec2:AuthorizeSecurityGroupIngress`
+* `ec2:CopyImage`
+* `ec2:CreateDhcpOptions`
+* `ec2:CreateInternetGateway`
+* `ec2:CreateNatGateway`
+* `ec2:CreateRoute`
+* `ec2:CreateRouteTable`
+* `ec2:CreateSecurityGroup`
+* `ec2:CreateSubnet`
+* `ec2:CreateTags`
+* `ec2:CreateVpc`
+* `ec2:CreateVpcEndpoint`
+* `ec2:CreateVolume`
+* `ec2:DeleteSnapshot`
+* `ec2:DeregisterImage`
```

Comment:
These last two are currently only used by `destroy cluster`.  We could use them to deregister our encrypted AMI copy once the bootstrap and control-plane machines came up, but we don't do that now.  So probably move them down into the *All permissions that are required to uninstall a cluster* section, at least for now.